app/uk/gov/hmrc/gform/auditing/AuditService.scala

```diff
@@ -41,21 +41,34 @@ trait AuditService {
       modelComponentId.toMongoIdentifier -> variadicValue.toSeq.mkString(",")
     }.toMap
 
+  def sendFormSaveEvent[D <: DataOrigin](
+    form: Form,
+    retrievals: MaterialisedRetrievals
+  )(implicit ec: ExecutionContext, hc: HeaderCarrier): Unit =
+    sendEvent("formSaved", form, Map.empty, retrievals, CustomerId.empty)
```

Comment:
Can we replace `save` with `started` to make it more clear that this is used only at the start of the form?